gr-fec/include/gnuradio/fec/polar_decoder_common.h

```diff
@@ -64,7 +70,7 @@ namespace gr {
         bool set_frame_size(unsigned int frame_size){return false;};
 
       private:
-        static const float D_LLR_FACTOR = -2.19722458f;
+        static _CONSTEXPR float D_LLR_FACTOR = -2.19722458f;
```

Comment:
I'm having difficulty with this with gcc 4.8.4 and boost 1.54., getting compilation syntax error on this line.  My guess is that for some reason the BOOST macro is not being defined, so the define becomes 'constexpr', which gcc 4.8.4 knows nothing about.

(Also, please submit bug fix pull requests off the 'maint' branch.)